### PR TITLE
mdd-binding-local-data-cell-activation

### PR DESCRIPTION
**Pull request by mddifilippo89**
Data-Grid BindingLocalData & CellActivation samples

### CODE REVIEW

File: samples/grids/data-grid/cell-activation/Pages/DataGridCellActivation.razor [hunk 1]
```diff
@@ -0,0 +1,40 @@
+﻿@page "/DataGridCellActivation"
+@* this namespace is required for Infragistics controls *@
+@using Infragistics.Blazor.Controls
+@inject IInfragisticsBlazor InfragisticsBlazor;
+@inject IJSRuntime JSRuntime;
+
+<div class="igContainer">
+
```
Comment (HUSSAR-mtrela) [inline samples/grids/data-grid/cell-activation/Pages/DataGridCellActivation.razor:8]: can you add a combobox to change ActivationMode property on the DataGrid?

File: samples/grids/data-grid/cell-activation/Pages/DataGridCellActivation.razor [hunk 2]
```diff
@@ -0,0 +1,40 @@
+﻿@page "/DataGridCellActivation"
+@* this namespace is required for Infragistics controls *@
+@using Infragistics.Blazor.Controls
+@inject IInfragisticsBlazor InfragisticsBlazor;
+@inject IJSRuntime JSRuntime;
```
Comment (HUSSAR-mtrela) [inline samples/grids/data-grid/cell-activation/Pages/DataGridCellActivation.razor:5]: remove ```@inject IJSRuntime JSRuntime;```

File: samples/grids/data-grid/cell-activation/Pages/DataGridCellActivation.razor [hunk 3]
```diff
@@ -0,0 +1,40 @@
+﻿@page "/DataGridCellActivation"
```
Comment (HUSSAR-mtrela) [inline samples/grids/data-grid/cell-activation/Pages/DataGridCellActivation.razor:1]: remove ```@page "/DataGridCellActivation"```

File: samples/grids/data-grid/binding-local-data/Pages/DataGridBindingLocalData.razor [hunk 4]
```diff
@@ -0,0 +1,133 @@
+﻿@page "/DataGridBindingLocalData"
+@* this namespace is required for Infragistics controls *@
+@using Infragistics.Blazor.Controls
+@inject IInfragisticsBlazor InfragisticsBlazor;
+@inject IJSRuntime JSRuntime;
```
Comment (HUSSAR-mtrela) [inline samples/grids/data-grid/binding-local-data/Pages/DataGridBindingLocalData.razor:5]: remove ```@Inject IJSRuntime JSRuntime;```

File: samples/grids/data-grid/cell-activation/Pages/DataGridCellActivation.razor [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+        {            
+            <DataGrid Height="100%" Width="100%"
+                      DataSource="@DataSource"
+                      AutoGenerateColumns="false"
+                      SelectionMode="GridSelectionMode.SingleCell"
+                      ActivationMode="GridActivationMode.Cell">
+                <TextColumn Field="Name" Width="170" />
+                <TextColumn Field="Street" HeaderText="Address" Width="150"/>
+                <TextColumn Field="City" Width="130"/>
+                <DateTimeColumn Field="Birthday" Width="150"/>
```
Comment (HUSSAR-mtrela) [inline samples/grids/data-grid/cell-activation/Pages/DataGridCellActivation.razor:21]: add 3-4 more columns to fill our some white space (see other samples that use EmployeeData)

**Review by HUSSAR-mtrela — CHANGES_REQUESTED** [review]

File: samples/grids/data-grid/cell-activation/Pages/DataGridCellActivation.razor (hunk 1 above)
Comment (mddifilippo89) [reply samples/grids/data-grid/cell-activation/Pages/DataGridCellActivation.razor:6]: we can revisit this, none of the other platforms don't have one.

**Review by HUSSAR-mtrela — APPROVED** [review]

**Pull request merged**